src/components/WorkflowForms/NewClientSignup/GatherInfo/components/ProvidersSection.jsx

```
@@ -0,0 +1,96 @@
+import strings from '../i18n-strings';
+
+const ProvidersSection = ({ providersState }) => {
+  const language = useContext(LanguageContext);
+  strings.setLanguage(language || mva_utils.get_device_language());
+
+  const {
+    selectedProviders,
+    availableProviders,
+    providerTypes,
+    selectedProvider,
+    setSelectedProvider,
```

ESLint

[error] 'setSelectedProvider' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)